File: src/ShitLeopard.Api/client-app/src/viewModels/DialogModel.ts
export class DialogModel {
  constructor(
    public id?: string,
    public dialogLineNumber?: number,
    public start?: string,
    public end?: string,
    public body?: string,
    public episodeNumber?: string,
    public episodeOffsetId?: string,
    public episodeTitle?: string,
    public seasonId?: string,
    public synopsis?: string
  ) {}
}

File: src/ShitLeopard.Common/Models/DialogModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ShitLeopard.Common.Models
{
    public class DialogModel
    {
        public string Id { get; set; }
        public long DialogLineNumber { get; set; }
        public string Start { get; set; }
        public string End { get; set; }
        public string Body { get; set; }
        public long EpisodeNumber { get; set; }
        public int? EpisodeOffsetId { get; set; }
        public string EpisodeTitle { get; set; }
        public string SeasonId { get; set; }
        public string Synopsis { get; set; }

        public string ShowName { get; set; }
    }
}
